app/views/customers/index.html.erb

```diff
@@ -1,18 +1,18 @@
-<h1>Listagem de clientes</h1>
+<h1><%= t('.title') %></h1>
 
 <table>
   <thread>
     <tr>
-      <th>Nome</th>
-      <th>CPF</th>
+      <th><%= t('.list.name') %></th>
+      <th><%= t('.list.cpf') %></th>
```

Comment:
Yes, but what do you suggest when I need to leave from the attribute's pattern? Like [this](https://github.com/rafaelrpbelo/car_dealership_sm/blob/internationalize-content/app/views/vehicles/index.html.erb#L8).
Using attribute's name I just can think about:
```html
...
<th><%= Vehicle.human_attribute(:brand) %></th>
<th><%= Vehicle.human_attribute(:version) %></th>
<th><%= t('.list.year_model') %></th>
...
```